egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1a_svd.sh

```diff
@@ -0,0 +1,361 @@
+#!/bin/bash
+# same as run_tdnn_opgru_1a.sh, but replacing Norm-OPGRU with LSTMP.
+# ./local/chain/compare_wer_general.sh --looped tdnn_lstm_1a_sp
+# System                tdnn_lstm_1a_sp
```

Comment:
It would be great if you show the WER information of the retrained SVD model here.